src/app.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const path = require('path');
+const exphbas = require("express-handlebars");
+const app = express();
+app.use(router)
+
+
+app.use(express.static(path.join(__dirname, '..', 'public')));
+
+app.set("views", path.join(__dirname, "views"));
```

Comment:
Please add disable x powered by. You're also missing (`app.use(express.json())`) and urlencoded.

Reply:
Double check the first code along.